lib/external_interaction/rubix/rubix_external_rest_api.dart

```diff
@@ -25,4 +28,26 @@ Future<Response> handleTransactionRequest(Request request) async {
   return Response.ok('Transaction request received');
 }
 
+Future<bool> externalApi({required String did,required String peerId,required String orgName,required String callBackUrl,required Token orgAccessToken })async {
+    var bodyJsonStr = jsonEncode({
+      'did': did,
+      'peerId': peerId,
+      'orgName': orgName,
+    });
+  var response = await http.post(
```

Comment:
we might need a try catch

Reply:
Ok will do that.

Reply:
http post can fail too, we need catch for that case

Reply:
Ok.